### PR TITLE
Key Gear: Tab

### PR DESCRIPTION
**Pull request by hedvigoscar**
Missing right now:

- GraphQL model classes (schema is not ready)
- Translations

We'll use this as a base for the rest of the feature implementation.

There are a lot of unfilled TODOs - these will be filled later

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/util/extensions/DimensionExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.hedvig.app.util.extensions
+
+import android.content.res.Resources
+
+val Int.dp: Int
```
Comment (FredrikAreschoug) [inline app/src/main/java/com/hedvig/app/util/extensions/DimensionExtensions.kt]: I don't love the name of this extension `toDp`?

File: app/src/main/java/com/hedvig/app/feature/keygear/KeyGearFragment.kt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+class KeyGearFragment : BaseTabFragment() {
+    override val layout = R.layout.fragment_key_gear
+
+    private val viewModel: KeyGearViewModel by sharedViewModel()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        items.adapter = KeyGearItemsAdapter()
+        items.addItemDecoration(GridSpacingItemDecoration(8.dp))
```
Comment (FredrikAreschoug) [inline app/src/main/java/com/hedvig/app/feature/keygear/KeyGearFragment.kt]: Maybe we should extract this constant to dimensions instead

File: app/src/main/java/com/hedvig/app/util/extensions/DimensionExtensions.kt (hunk 1 above)
Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/util/extensions/DimensionExtensions.kt]: I think it's quite a nice name - I stole it from some Jetpack Compose-code that used this type of extension.

File: app/src/main/java/com/hedvig/app/feature/keygear/KeyGearFragment.kt (hunk 2 above)
Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/keygear/KeyGearFragment.kt]: I see your point - perhaps we could use a kotlin-based `const val`?

**Pull request merged**